docs/user_guide/02_medrecord_intro.md

```diff
@@ -0,0 +1,239 @@
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 46
+---
+```
+
+This will expand the MedRecord, adding several new nodes to the graph. However, these nodes are not yet connected, so let's establish relationships between them!
+
+```{image} https://raw.githubusercontent.com/limebit/medmodels-static/main/imgs/user_guide/02/02_medrecord_intro_02.png
+:class: transparent-image
+```
+## Adding Edges to a MedRecord
+
+To capture meaningful relationships between nodes, such as linking patients to prescribed medications, we add edges to the MedRecord. These edges must be specified in a relation table, such as the one shown below:
+
+:::{list-table} Patient-Medication Relation
+:widths: 15 15 15
+:header-rows: 1
+
+*   - Pat_ID
+    - Med_ID
+    - time
+*   - Patient 02
+    - Med 01
+    - 2020/06/07
+*   - Patient 02
+    - Med 02
+    - 2018/02/02
+*   - Patient 03
+    - Med 02
+    - 2019/03/02
+:::
+
+We can add these edges then to our MedRecord Graph:
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 54
+---
+```
+This results in an enlarged Graph with more information.
+
+```{image} https://raw.githubusercontent.com/limebit/medmodels-static/main/imgs/user_guide/02/02_medrecord_intro_03b.png
+:class: transparent-image
+```
+## Adding Groups to a MedRecord
+
+For certain analyses, we may want to define specific subcohorts within our MedRecord for easier access. We can do this by defining named groups withing our MedRecored.
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 58
+---
+```
+This group will include all the defined nodes, allowing for easier access during complex analyses. Both nodes and edges can be added to a group, with no limitations on group size. Additionally, nodes and edges can belong to multiple groups without restriction.
+
+```{image} https://raw.githubusercontent.com/limebit/medmodels-static/main/imgs/user_guide/02/02_medrecord_intro_04.png
+:class: transparent-image
+```
+
+## Saving and Loading MedRecords
+
+When building a MedRecord, you may want to save it to create a persistent version. This can be done by storing it as a RON (Rusty Object Notation) file. The MedRecord can then be reloaded, allowing you to create a new instance from the saved RON file.
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 80-81
+---
+```
+
+## Printing Overview Tables
+
+The MedModels MedRecord class is designed to efficiently handle large datasets while maintaining a standardized data structure that supports complex analysis methods. As a result, the structure within the MedRecord can become intricate and difficult to manage. To address this, MedModels offers tools to help keep track of the graph-based data. One such tool is the `print_node_overview` method:
```

Comment:
We are referring to the method as `print_node_overview` but then we call a different one.

Reply:
Function will be renamed